src/ray/raylet/scheduling/cluster_task_manager.h

```diff
@@ -92,6 +114,8 @@ class ClusterTaskManager {
   std::deque<Work> tasks_to_dispatch_;
   /// Tasks waiting for arguments to be transferred locally.
   absl::flat_hash_map<TaskID, Work> waiting_tasks_;
+  /// Tasks that are currently running.
```

Comment:
Can you update the comment to explain why we need to keep track of these?

Reply:
This was originally in order to replicate the same behavior as the old scheduler's task cancellation behavior (return true if the task is already running). I thought it might still be useful later? I'm also open to removing it.

Reply:
It doesn't seem like it's ever read. There is a method reading it, but it never gets called.

Let's remove it; we should try not to add code that might be useful later since it often becomes cruft.